bucket_99/rofi/specification

```diff
@@ -0,0 +1,47 @@
+# DEF[PORTVERSION]=	1.5.1
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		rofi
+VERSION=		1.5.1
+KEYWORDS=		x11
+VARIANTS=		standard
+SDESC[standard]=	Window switcher, run dialog and dmenu replacement
+HOMEPAGE=		https://github.com/DaveDavenport/rofi
+CONTACT=		nobody
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		https://github.com/DaveDavenport/rofi/releases/download/1.5.1
+DISTFILE[1]=		x-1.5.1.tar.gz:main
+
+SPKGS[standard]=	single
+
+OPTIONS_AVAILABLE=	none
+OPTIONS_STANDARD=	none
+
+LICENSE=		MIT:single
+LICENSE_FILE=		MIT:{{WRKSRC}}/COPYING
+
+FPC_EQUIVALENT=		x11/rofi
+
+BUILD_DEPENDS=		check:single:standard
+			flex:single:standard
+
+BUILDRUN_DEPENDS=	startup-notification:single:standard
+			xorg-xcb-util:single:standard
+			xorg-xcb-util-wm:single:standard
+			xorg-xcb-util-xrm:single:standard
+			libxkbcommon:single:standard
+
+USES=			bison gmake libtool pkgconfig shebangfix
+GNOME_COMPONENTS=	cairo gdkpixbuf librsvg pango
+XORG_COMPONENTS=	xcb
+MUST_CONFIGURE=		gnu
+CONFIGURE_ENV=		LEX="{{LOCALBASE}}/bin/flex"
+SHEBANG_FILES=		script/get_git_rev.sh
+			script/rofi-sensible-terminal
+bash_CMD=		/bin/sh
+
+post-patch:
+# Fix rofi-theme-selector which assumes we have GNU sed
```

Comment:
You probably want to start this line with a hard tab.
Comments that start on column 1 are stripped out of the buildsheet, but tabbed comments (on make targets) are kept, so it depends on whether you want the comment to be retained on the final build sheet.  In this case, I would think you would.

Reply:
Fine.

`bash_CMD= /bin/sh` is also giving a problem: 

`Error : Failed to parse specification
Line 42: Parse failed, content unrecognized.`

The manual is very succinct and I can't get what is wrong.

Reply:
check shebangfix.mk in conspiracy/Mk/Uses
also https://github.com/jrmarino/Ravenports/wiki/Chapter-10#shebangfix
syntax differs from FreeBSD